src/components/Button.js

```diff
@@ -8,7 +8,7 @@ const Button = styled.button`
   margin: 0 1px 1px 0;
   font-weight: 300px;
   box-shadow: 0px 8px 15px var(--color-light-shadow);
-  background-color: ${props => (props.isFavorite ? '#d3d3d3' : 'white')};
+  background-color: white;
```

Comment:
maybe you can change this to a variable like `--color-basis` then you can switch easier to different color schemes